openhtf/util/monitors.py

```diff
@@ -108,7 +108,7 @@ def Wrapper(phase_func):
     @plugs.plug(update_kwargs=False, **monitor_plugs)
     @measurements.measures(
         measurements.Measurement(measurement_name).WithUnits(
-            units).WithDimensions(uom.UOM['MILLISECOND']))
+            units).WithDimensions(uom.MILLISECOND))
```

Comment:
i went with millis because all our other timestamps are in millis (`start_time_millis`, `end_time_millis`, etc.).  I'm not particularly attached to it either way.